shared/fs/header.js

```diff
@@ -0,0 +1,167 @@
+// @flow
+import * as React from 'react'
+import * as Types from '../constants/types/fs'
+import {globalStyles, globalColors, globalMargins, isMobile} from '../styles'
+import {Avatar, Box, ClickableBox, Icon, Text} from '../common-adapters'
+import {FolderHeaderBreadcrumbConnector} from './header-container'
+
+type FolderHeaderProps = {
+  path: Types.Path,
+}
+
+const stylesCommonRow = {
```

Comment:
we usually put the styles at the bottom so when you open the file you see the components

Reply:
lint yells at me with the `no-use-before-define` complaint.

Reply:
NVmind that warning went away after I made it functional